test/Npgsql.Tests/ReaderTests.cs

```diff
@@ -1111,6 +1110,53 @@ public async Task NullableScalar()
             }
         }
 
+        [Test, Description("Times out an async operation, testing that cancellation occurs successfully")]
+        public async Task ReaderTimeoutAsyncSoft()
+        {
+            if (IsMultiplexing)
+                return; // Multiplexing, cancellation
+
+#if NET461
+            Assert.Ignore("Async timeouts aren't supported with .net framework");
```

Comment:
nit: Assert.Ignore emits a "skip" message in the tests - we should only do this if the test is temporarily skipped, and we want to get back to it and fix it (eventually). Cases which will never work such as this one should simply return, making the test pass successfully.

Reply:
Sure, conditional compilation is also fine. The only point here is not to have a "test skipped" message every time we run the tests, for something that will never be implemented.